newIDE/app/src/MainFrame/EditorContainers/ExternalEventsEditorContainer.js

```diff
@@ -180,4 +181,11 @@ export class ExternalEventsEditorContainer extends React.Component<
 
 export const renderExternalEventsEditorContainer = (
   props: RenderEditorContainerPropsWithRef
-) => <ExternalEventsEditorContainer {...props} />;
+) =>
+  props.project ? (
```

Comment:
Is not having a project even possible?

Reply:
I think in practice it would not be possible, but `RenderEditorContainerPropsWithRef` is a common type for all editor tab contents that do not necessarily need a project to be opened (like extensions)

Reply:
In this case, do a: 

```
    if (!project) {
      //TODO: Error component
      return <div>No project opened!</div>;
    }
```

This "TODO" is there since the beginning of the app... but well it's never shown, it's just to give a hint to a future developer that would rework the main frame and forget to pass the `project`.